<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_MainWid</class>
 <widget class="QWidget" name="Home_MainWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>743</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>位置信息</string>
     </property>
     <widget class="QLineEdit" name="roomEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>200</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>49</y>
        <width>50</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>机房:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="locatEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>200</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>50</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>位置:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>网络地址</string>
     </property>
     <widget class="QLineEdit" name="ipEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>IP地址:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>子网掩码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="maskEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>DHCP：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="dhcpBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>禁用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>启用</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="gwEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>默认网关:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="dnsEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>DNS服务器:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Web登陆</string>
     </property>
     <widget class="QLineEdit" name="pwdEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>106</y>
        <width>200</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>106</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>账号:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="usrEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>200</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>设备模式</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>50</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>模式：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="modeBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>180</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>标准</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>级联</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>双电源</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Modbus-RTU</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>106</y>
        <width>50</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="addrBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>106</y>
        <width>180</width>
        <height>26</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>集成接口</string>
     </property>
     <widget class="QComboBox" name="modbusBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>禁用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TCP-Modbus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTU-Modbus</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Modbus：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="rpcBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>禁用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JSON-RPC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XML-RPC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>远程调用：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="pushBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>禁用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UDP推送</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>http推送</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>主动推送：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>推送地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="portEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>推送端口：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="ctrlBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>禁用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>启用</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>设备反控：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>操作区</string>
     </property>
     <widget class="QPushButton" name="startBtn">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>88</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>提 交</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>171</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>自动发现</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="resetCheck">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置后自动重启</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>指定IP地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dstEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
